src/less/combobox/base/combobox.less

```diff
@@ -15,6 +30,42 @@ span.combobox__icon {
     top: 0;
 }
 
+.combobox__options--fix-width[role="listbox"] {
+    width: 100%;
+}
+
+.combobox__options--reverse[role="listbox"] {
+    right: 0;
+}
+
+.combobox__listbox {
```

Comment:
What necessitates the new `.combobox__listbox` wrapper for the options? Why the extra `<div>`?

Reply:
Good question. I'm following our pattern of each module needing a root element. Given that the listbox inside of listbox button is now essentially a nested module, I've found that it is a huge help in the javascript layer to have this additional "root" element for a nested module too. Plus, it gives us a little extra styling flexibility (in terms of padding/margin) and also can be used as a aria-live hook if we need.

Reply:
So, it's a root for just a "listbox"?

Also, what other styling is needed? I don't see any styling changes except that it is now the element that gets set to `display: block`.

Reply:
Correct. It's the root for a listbox. If you look through the js files it might make more sense. No other styling needed currently.